src/main/kotlin/model/ModelFacade.kt

```diff
@@ -23,7 +23,8 @@ class ModelFacade {
 
     companion object {
         private val log: Logger = Logger.getInstance(ModelFacade::class.java)
-        private val beamSearchModule = org.tensorflow.TensorFlow.loadLibrary(getPathToBeamModule());
+// Throws exception on Windows. Though astrid still doesn't work without it :( TensorFlow can't load the model.
+//        private val beamSearchModule = org.tensorflow.TensorFlow.loadLibrary(getPathToBeamModule());
```

Comment:
I doubt that handling the issue this way is a good idea. If the plugin could work on Linux and Mac we should preserve this. A possible solution here - to hide the TF implementation under `PredictionModel` that can be safely created by the `Downloader.buildModel() -> PredictionModel?` method. In the case of windows just return `null` and handle it inside `ModelFacade`.